<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
  <testsuite name="fastlane.lanes">
    
    
    
      
      <testcase classname="fastlane.lanes" name="0: import_from_git" time="1.492281">
        
      </testcase>
    
      
      <testcase classname="fastlane.lanes" name="1: git_pull" time="2.234439">
        
      </testcase>
    
      
      <testcase classname="fastlane.lanes" name="2: ensure_git_branch" time="0.019293">
        
      </testcase>
    
      
      <testcase classname="fastlane.lanes" name="3: version_bump_podspec" time="0.002823">
        
      </testcase>
    
      
      <testcase classname="fastlane.lanes" name="4: git_add" time="0.027455">
        
      </testcase>
    
      
      <testcase classname="fastlane.lanes" name="5: git_commit" time="0.048277">
        
      </testcase>
    
      
      <testcase classname="fastlane.lanes" name="6: push_to_git_remote" time="1.097088">
        
          <failure message="/Users/uxiu/.rvm/gems/ruby-3.0.0/gems/fastlane-2.193.1/fastlane/lib/fastlane/actions/actions_helper.rb:67:in `execute_action&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/gems/fastlane-2.193.1/fastlane/lib/fastlane/runner.rb:255:in `block in execute_action&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/gems/fastlane-2.193.1/fastlane/lib/fastlane/runner.rb:229:in `chdir&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/gems/fastlane-2.193.1/fastlane/lib/fastlane/runner.rb:229:in `execute_action&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/gems/fastlane-2.193.1/fastlane/lib/fastlane/runner.rb:157:in `trigger_action_by_name&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/gems/fastlane-2.193.1/fastlane/lib/fastlane/fast_file.rb:159:in `method_missing&apos;&#10;../../../../../../var/folders/9w/2mc3w3y57m50r0w86nkqcqw80000gn/T/fl_clone20210907-79925-aocq83/fastlane-files/fastlane/Fastfile:34:in `block in parsing_binding&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/gems/fastlane-2.193.1/fastlane/lib/fastlane/lane.rb:33:in `call&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/gems/fastlane-2.193.1/fastlane/lib/fastlane/runner.rb:49:in `block in execute&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/gems/fastlane-2.193.1/fastlane/lib/fastlane/runner.rb:45:in `chdir&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/gems/fastlane-2.193.1/fastlane/lib/fastlane/runner.rb:45:in `execute&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/gems/fastlane-2.193.1/fastlane/lib/fastlane/lane_manager.rb:47:in `cruise_lane&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/gems/fastlane-2.193.1/fastlane/lib/fastlane/command_line_handler.rb:36:in `handle&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/gems/fastlane-2.193.1/fastlane/lib/fastlane/commands_generator.rb:109:in `block (2 levels) in run&apos;&#10;/Users/uxiu/.rvm/rubies/ruby-3.0.0/lib/ruby/gems/3.0.0/gems/commander-4.6.0/lib/commander/command.rb:187:in `call&apos;&#10;/Users/uxiu/.rvm/rubies/ruby-3.0.0/lib/ruby/gems/3.0.0/gems/commander-4.6.0/lib/commander/command.rb:157:in `run&apos;&#10;/Users/uxiu/.rvm/rubies/ruby-3.0.0/lib/ruby/gems/3.0.0/gems/commander-4.6.0/lib/commander/runner.rb:444:in `run_active_command&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/gems/fastlane-2.193.1/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:117:in `run!&apos;&#10;/Users/uxiu/.rvm/rubies/ruby-3.0.0/lib/ruby/gems/3.0.0/gems/commander-4.6.0/lib/commander/delegates.rb:18:in `run!&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/gems/fastlane-2.193.1/fastlane/lib/fastlane/commands_generator.rb:353:in `run&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/gems/fastlane-2.193.1/fastlane/lib/fastlane/commands_generator.rb:42:in `start&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/gems/fastlane-2.193.1/fastlane/lib/fastlane/cli_tools_distributor.rb:122:in `take_off&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/gems/fastlane-2.193.1/bin/fastlane:23:in `&lt;top (required)&gt;&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/bin/fastlane:23:in `load&apos;&#10;/Users/uxiu/.rvm/gems/ruby-3.0.0/bin/fastlane:23:in `&lt;main&gt;&apos;&#10;&#10;Exit status of command &apos;git push origin master:master --tags&apos; was 128 instead of 0.&#10;remote: Permission to UUKit/UUKit.git denied to uxiume.&#10;fatal: unable to access &apos;https://github.com/UUKit/UUKit.git/&apos;: The requested URL returned error: 403&#10;" />
        
      </testcase>
    
  </testsuite>
</testsuites>
